app/javascript/mastodon/locales/fr.json

```diff
@@ -55,8 +55,6 @@
   "confirmations.domain_block.message": "Êtes-vous vraiment, vraiment sûr⋅e de vouloir bloquer {domain} en entier ? Dans la plupart des cas, quelques blocages ou masquages ciblés sont suffisants et préférables.",
   "confirmations.mute.confirm": "Masquer",
   "confirmations.mute.message": "Confirmez vous le masquage de {name} ?",
-  "confirmations.unfollow.confirm": "Unfollow",
-  "confirmations.unfollow.message": "Are you sure you want to unfollow {name}?",
```

Comment:
@ykzts the `yarn manage:translations` removed those strings automatically so I figured out that they must have been removed from the default locale. I can bring them back if it's an issue.